### PR TITLE
Feat/119/update logger

### LINKED ISSUE #119: [FEATURE/CHORE] Update console log statements to a logger
**Is your feature request related to a problem? Please describe.**
Right now all logs are handled by `console.log` or similar.

**Describe the solution you'd like**
We should migrate all of these calls to using something like [winston](https://github.com/winstonjs/winston). This will allow us to be more flexible while logging and getting more context. (such as information on who did what)

**Describe alternatives you've considered**
We can stick with our current logging system, but it doesn't scale or give enough context often.

**Additional context**
This process should be done slowly, and can be spread across multiple PR requests.

### PR DESCRIPTION
**Pull request by bradtaniguchi**
**Description:**
Refactored all `console` calls to a winston logger.

Closes #119 

**note** this is a draft because the logging format isn't finalized yet, its currently just the default.

### CODE REVIEW

**Comment by bradtaniguchi** [comment]
I'll try to get to this tomorrow (setting a calendar reminder ;D)

File: src/index.ts [hunk 1]
```diff
@@ -39,7 +40,7 @@ const client = new Client({ partials: ['MESSAGE', 'CHANNEL', 'REACTION'] });
     if (config.VERBOSE) {
       // if we are to print each message as is.
       client.on('message', (message) => {
-        console.log('>> ', message.content);
+        logger.silly('>> ', message.content);
```
Comment (naomi-lgbt) [inline src/index.ts:43]: `logger.silly`? 🤔 😛

Comment (bradtaniguchi) [reply src/index.ts:43]: `silly` is the lowest log level, so it wont be outputted to the console if winston is configured with **any** higher log level.

**Comment by bradtaniguchi** [comment]
@nhcarrigan Is this ok to merge?

**Review by naomi-lgbt — APPROVED** [review]
This looks good to me!

**Pull request merged**